can/interfaces/vector/canlib.py

```diff
@@ -0,0 +1,176 @@
+# -*- coding: utf-8 -*-
+"""
+Ctypes wrapper module for Vector CAN Interface on win32/win64 systems
+Authors: Julien Grave <grave.jul@gmail.com>, Christian Sandberg
+"""
+# Import Standard Python Modules
+# ==============================
+import ctypes
+import logging
+import sys
+import time
+
+# Import Modules
+# ==============
+from can import BusABC, Message
+
+# Define Module Logger
+# ====================
+LOG = logging.getLogger('can.vector')
```

Comment:
Perhaps, pylint actually complains if it is lowercase since it is a module constant...